sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -236,9 +239,9 @@ def from_connection_string(
          specified Topic the client connects to.
         :keyword mode: The mode with which messages will be retrieved from the entity. The two options
          are PeekLock and ReceiveAndDelete. Messages received with PeekLock must be settled within a given
-         lock period before they will be removed from the queue. Messages received with ReceiveAndDelete
-         will be immediately removed from the queue, and cannot be subsequently rejected or re-received if
-         the client fails to process the message. The default mode is PeekLock.
+         lock period (or have their lock renewed) before they will be removed from the queue. Messages received 
+         with ReceiveAndDelete will be immediately removed from the queue, and cannot be subsequently rejected 
+         or re-received if the client fails to process the message. The default mode is PeekLock.
```

Comment:
```suggestion
         or re-received if the client fails to process the message. 
         
         The default mode is PeekLock.
```
(can you do newlines? Might help this sentence to stand out)

Reply:
I'm hesitant only about putting a linebreak halfway through the block with no other linebreaks.  We actually wanted to space all of these out, but that'll likely be a unified effort so this would be less discontinuous then; in leiu of going all the way though I do like the thought of accenting it more by having a single \n.

```suggestion
         or re-received if the client fails to process the message.      
         The default mode is PeekLock.
```

Reply:
That works just as well, I think. We've got to work within the constraints of the tooling :)